Makefile

```diff
@@ -33,79 +36,56 @@ verify:
 unprepublish:
 	sed s/"dist\/"/"src\/"/ packages/tc-ui/package.json > tmp && mv tmp packages/tc-ui/package.json
 
+clean-deps: unprepublish
+	rm -rf packages/*/node_modules
```

Comment:
Does this need to clean up lock files in the packages too?

Reply:
nope - the install step for packages skips package-lock https://github.com/Financial-Times/treecreeper/blob/master/package.json#L59 (following your advice from a few months ago)